lua/mini/jump.lua

```diff
@@ -7,7 +7,7 @@
 --- Features:
 --- - Extend f, F, t, T to work on multiple lines.
 --- - Repeat jump by pressing f, F, t, T again. It is reset when cursor moved
----   as a result of not jumping.
+---   as a result of not jumping or timeout after idle time (duration customizable).
```

Comment:
This is better to be reformatted (`customizable).` on new line) to fit in 78 width when documentation is regenerated.